docs/mastering-plone/installation.md

```diff
@@ -188,6 +188,8 @@ If successful, you get:
 
 > 🎭 Volto started at <http://localhost:3000> 🚀
 
+**Troubleshooting** [^troubleshooting]
```

Comment:
I first thought that this was a sub-heading, like `**Create your Volto project:**` above, and did not think to click the link. I assumed that the following content _was_ troubleshooting.

Suggest this markup.

````suggestion
```{seealso}
Something not right? See **Troubleshooting**.[^troubleshooting]
```
````